src/components/Feedback.js

```diff
@@ -0,0 +1,91 @@
+import { h } from 'preact'
+import { useState, useRef, useEffect } from 'preact/hooks'
+import { useI18n, useSoftkey, useNavigation } from 'hooks'
+import { sendFeedback } from 'utils'
+
+export const Feedback = ({ close }) => {
+  const containerRef = useRef()
+  const i18n = useI18n()
+  const [message, setMessage] = useState()
+  const [validInput, setValidInput] = useState(null)
+  const [, setNavigation, getCurrent] = useNavigation('Feedback', containerRef, containerRef, 'y')
+
+  const items = [
+    { text: `<a data-selectable>${i18n('feedback-terms-of-survey')}</a>`, link: 'https://wikimediafoundation.org/' }, // TODO: link not provided yet
+    { text: `<a data-selectable>${i18n('feedback-privacy-policy')}</a>`, link: 'https://foundation.m.wikimedia.org/wiki/Privacy_policy' },
+    { text: `<a data-selectable>${i18n('feedback-terms-of-use')}</a>`, link: 'https://foundation.m.wikimedia.org/wiki/Terms_of_Use/en' }
+  ]
+  const hyperlinks = items.map(i => i.text)
+
+  const onKeyRight = () => {
+    if (message) {
+      sendFeedback(message)
```

Comment:
multiple event sent after multiple time press of right key, should we lock or delay or throttle the event sent?

Reply:
Yea that's not a bad idea. Just curious, I'm not too familiar with `EventLogging` yet, would multiple send events simply mean that we would get repeated events on the kibana dashboard?

Reply:
We have to make sure users cannot submit the same feedback more than once by mistake. We could clear the form after the submission.

This feedback is private and won't show on any kibana dashboard. We'll work with Neil to export the feedback to a spreadsheet (or something) regularly for Angie and a few others to consider.

Reply:
> clear the form after the submission

This sounds good instead of delay or throttle implementation!